sympy/matrices/determinant.py

```diff
@@ -832,7 +926,7 @@ def _minor(M, i, j, method="berkowitz"):
 
     method : string, optional
         Method to use to find the determinant of the submatrix, can be
-        "bareiss", "berkowitz" or "lu".
+        "bareiss", "berkowitz", "laplace" or "lu".
```

Comment:
missing "bird"?

Reply:
Yes, thanks for spotting. Addressed in a777540c78ce8da5ca9e68d4182f68b6d299a231